tests/includes/storage.sh

```diff
@@ -47,3 +54,51 @@ unit_state() {
 
 	echo ".storage[\"$app_name/$app_index\"] | .attachments | .units[\"$unit_name/$unit_index\"][\"life\"]"
 }
+
+# like wait_for but for storage formats. Used to wait for a certain condition in charm storage.
+wait_for_storage() {
```

Comment:
maybe we need to move this function to `includes/wait_for.sh`?

Reply:
I thought since everything related to storage is in storage.sh. It would be nice to just put it there. Also wait_for_storage uses `juju storage` unlike `wait_for` which uses `juju status`.